Robot_Adapter/Types/Comparer.cs

```diff
@@ -47,6 +48,7 @@ protected void SetupComparers()
                 {typeof(Bar), new BarEndNodesDistanceComparer(3) },
                 {typeof(ISectionProperty), new BHoMObjectNameOrToStringComparer() },
                 {typeof(Material), new BHoMObjectNameComparer() },
```

Comment:
ohhh, wonder how this did not get picked up before!
Would get rid of the Material one here, as physical/common materials are not supported.